vhdbuilder/packer/install-dependencies.sh

```diff
@@ -370,23 +370,23 @@ K8S_VERSIONS="
 1.16.4
 1.16.1
 1.14.6_v0.0.5
-1.14.7-hotfix.20200326
-1.14.8-hotfix.20200326
+1.14.7-hotfix.20200408.1
+1.14.8-hotfix.20200408.1
 1.15.3_v0.0.5
 1.15.4_v0.0.5
 1.15.5_f0.0.2
 1.15.7-hotfix.20200326
-1.15.10-hotfix.20200326
```

Comment:
Need to keep these old image until we upgraded all ccp chart to use the new images